projects/admin/src/app/menu/menu-definition/menu-app.ts

```diff
@@ -215,7 +215,16 @@ export const MENU_APP: IMenuParent[] = [
         access: {
           permissions: [PERMISSIONS.PTTR_ACCESS]
         }
-      }
+      },
+      {
+        name: 'Report Configuration',
+        router_link: ['/', 'records', 'stats_cfg'],
+        attributes: { id: 'stats-cfg-menu' },
+        extras: { iconClass: 'fa fa-cog' },
```

Comment:
`fa-bar-chart` would be nice?

Reply:
`fa-bar-chart` is already used by the main menu.